symengine/lib/symengine_wrapper.pyx

```diff
@@ -216,6 +230,11 @@ def sympy2symengine(a, raise_error=False):
         return DenseMatrix(row, col, v)
     elif isinstance(a, sympy.polys.domains.modularinteger.ModularInteger):
         return PyNumber(a, sympy_module)
+    elif sympy.__version__ > sympy_version:
```

Comment:
Can you replace `sympy_version` with `1.0`? Having it as a global variable makes no sense